src/app/auth/logged-user.service.ts

```diff
@@ -20,6 +20,7 @@ import { takeUntilDestroyed } from "@angular/core/rxjs-interop";
 export class LoggedUserService extends UnsubscribeDestroyRefAdapter {
     private loggedInUser: MaybeNull<AccountFragment> = null;
     private loggedInUser$: Subject<MaybeNull<AccountFragment>> = new ReplaySubject<MaybeNull<AccountFragment>>(1);
+    private adminPriveleges$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
```

Comment:
Correct spelling is "privilege", correct everywhere.

Reply:
Fixed